__tests__/topics.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("News API", () => {
+  describe("Invalid Paths", () => {
+    test("repsonds with a status of 404 when given an invalid path", () => {
+      return request(app)
+        .get("/api/notapath")
+        .expect(404)
+        .then(({body:{message}}) => {
+          expect(message).toBe("Invalid API path request");
+        });
+    });
+  });
+
+  describe("GET /api/topics", () => {
+    test("responds with a status of 200 and returns topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(topics).toBeInstanceOf(Array);
```

Comment:
strictly speaking the instanceOf assertion is not adding anything here because it if has length and works inside a forEach loop then it must be an array - but you can keep it if you prefer